integration/src/binance/binance.ts

```diff
@@ -65,18 +68,63 @@ export function binanceTests(get: () => { wallet: HDWallet; info: HDWalletInfo }
           tx: tx02_unsigned,
           addressNList: bip32ToAddressNList("m/44'/714'/0'/0/0"),
           chain_id: "Binance-Chain-Nile",
-          account_number: "24250",
-          sequence: 0,
```

Comment:
Note that these values were previously being ignored due to a bug in hdwallet-native.